es6-shim.js

```diff
@@ -69,7 +69,7 @@
   var _some = Function.call.bind(Array.prototype.some);
 
   var defineProperty = function (object, name, value, force) {
-    if (!force && name in object) { return; }
+    if (!object || (!force && name in object)) { return; }
```

Comment:
do we need this change now? if we had to keep the object check, i'd prefer to make it throw, so the error can be surfaced quickly, but instead i'd rather count on always calling `defineProperty` correctly.

Reply:
I think it is okay to revert this.